tools/azsdk-cli/Azure.Sdk.Tools.Cli/.tspupdate-sessions/f016b5a9c63e4fe6b40be457f847fb62.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Do we need a git ignore for this or is it intentional? If so we can we move it into a test assets folder or something?

Reply:
https://github.com/Azure/azure-sdk-tools/tree/main/tools/azsdk-cli/Azure.Sdk.Tools.Cli.Tests/TestAssets